Marlin/Configuration_adv.h

```diff
@@ -232,7 +232,13 @@
 #define INVERT_E_STEP_PIN false
 
 // Default stepper release if idle. Set to 0 to deactivate.
+// Steppers will shut down DEFAULT_STEPPER_DEACTIVE_TIME seconds after the last move when DISABLE_INACTIV_? is true.
+// Time can be set by M18 and M84.
 #define DEFAULT_STEPPER_DEACTIVE_TIME 60
+#define DISABLE_INACTIV_X true
+#define DISABLE_INACTIV_Y true
+#define DISABLE_INACTIV_Z true  // set to false if the nozzle will fall down on your printed part when print has finished.
+#define DISABLE_INACTIV_E true
```

Comment:
Rename these as `DISABLE_INACTIVE_*` and I think this PR can be moved forward. It addresses two behaviors that are somewhat related. I guess the `axis_homed` stuff conflicts with my recent PR on that subject. I suppose this PR should precede that one, because mine is smaller.